src/main/java/SQL_createTable/SQL.java

```diff
@@ -0,0 +1,169 @@
+            connection = getConnection(URL, LOGIN, PASSWORD);
+            connection.setAutoCommit(false);
+            stm = connection.createStatement();
+            String sql = "INSERT INTO author (id, name ,surname) VALUES (1,'Сергей','Есенин')," +
+                    "(2,'Владимир','Высоцкий')," +
+                    "(3,'Александр','Пушкин')";
+            stm.executeUpdate(sql);
+            connection.commit();
+        } catch (SQLException | RuntimeException e) {
+            System.err.println(e);
+            logger.error(e);
+        } finally {
+            connection.close();
+            stm.close();
+            logger.info("Resources are closed.");
+        }
+    }
+
+    private static void insertValuesIntoTableBook() throws SQLException {
+        Connection connection = null;
+        Statement stm = null;
+        try {
+            connection = getConnection(URL, LOGIN, PASSWORD);
+            connection.setAutoCommit(false);
+            stm = connection.createStatement();
+            String sql = "INSERT INTO book (id, title, author_id, pages) VALUES (1,'Письмо к женщине',1,100)," +
+                    "(2,'Роман о девочках',2,132)," +
+                    "(3,'Зимнее утро',3,245)";
+            stm.executeUpdate(sql);
+            connection.commit();
+        } catch (SQLException | RuntimeException e) {
+            System.err.println(e);
+            logger.error(e);
+        } finally {
+            connection.close();
+            stm.close();
+            logger.info("Resources are closed.");
+        }
+    }
+
+    private static void displayBookPages() throws SQLException {
+        Connection connection = null;
+        Statement stm = null;
+        ResultSet rs = null;
+        try {
+            connection = getConnection(URL, LOGIN, PASSWORD);
+            connection.setAutoCommit(false);
+            stm = connection.createStatement();
+            rs = stm.executeQuery("select title AS Book, pages AS Pages from book\n" +
+                    "where pages > 100");
+            while (rs.next()) {
+                String str = rs.getString("Book") + " " + rs.getString("Pages");
+                System.out.println(str);
+            }
+        } catch (SQLException | RuntimeException e) {
+            System.err.println(e);
+            logger.error(e);
+        } finally {
+            rs.close();
+            stm.close();
+            connection.close();
+            logger.info("Resources are closed.");
+        }
+    }
+
+    private static void displayAuthorAndBooksWherePagesMore300() throws SQLException {
+        Connection connection = null;
+        Statement stm = null;
+        ResultSet rs = null;
+        try {
+            connection = getConnection(URL, LOGIN, PASSWORD);
+            connection.setAutoCommit(false);
+            stm = connection.createStatement();
+            rs = stm.executeQuery("select a1.name AS Author, a1.surname, a2.title AS Book from author a1\n" +
```

Comment:
How about author name uniquness?
If we have more than 1 book, we will get more than 1 name (as a result of the select we need to get only author names/surnames)